src/Apps/Artwork/routes.tsx

```diff
@@ -12,8 +14,35 @@ export const routes = [
         }
       }
     `,
-    cacheConfig: {
-      force: true,
+
+    /**
+     * Disable artwork page caching for everything but *inquiry works*
+     */
+    getCacheConfig: async ({ params }) => {
```

Comment:
Does this wind up adding a blocking req (I'm guessing not) to each artwork route navigation?

Reply:
btw, we don't support `async` functions for `getCacheConfig`, so it's possible that the use of `async` here is itself causing problems